mvc/src/main/java/com/interface21/webmvc/servlet/mvc/tobe/adapter/HandlerExecutionHandlerAdapter.java

```diff
@@ -0,0 +1,20 @@
+package com.interface21.webmvc.servlet.mvc.tobe.adapter;
+
+import com.interface21.webmvc.servlet.ModelAndView;
+import com.interface21.webmvc.servlet.mvc.tobe.HandlerExecution;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+
+public class HandlerExecutionHandlerAdapter implements HandlerAdapter {
+
+    @Override
+    public boolean supports(Object handler) {
+        return handler instanceof HandlerExecution;
+    }
+
+    @Override
+    public ModelAndView handle(HttpServletRequest request, HttpServletResponse response, Object handler)
+            throws Exception {
+        return ((HandlerExecution) handler).handle(request, response);
```

Comment:
변수로 한번 추출해주어도 좋을 것 같아요

```suggestion
       HandlerExecution handlerExecution = (HandlerExecution) handler;
        return handlerExecution.handle(request, response);
```

Reply:
[반영] [ee014be](https://github.com/woowacourse/java-mvc/pull/780/commits/ee014be1298bfefe94b406913fe677e74f055753)